minerl/data/data_pipeline.py

```diff
@@ -405,7 +407,11 @@ def traj_iter():
 
             jobs = [(f, -1, None)
                     for f in self._get_all_valid_recordings(self.data_dir)]
+            if epoch_size is not None and len(jobs) < epoch_size:
+                raise ValueError("Set epoch size to {epoch_size} "
+                                 " but there are only {len(jobs)} jobs available.")
             np.random.shuffle(jobs)
+            jobs = jobs[:epoch_size]
```

Comment:
I was under the impression we were going to prevent the data from being loaded in the first place? (Possibly Cody did this in a different PR?)

Reply:
Hmmm I think that this is being confused with some other fix that Cody made. I'll ask her about this in the meeting tomorrow.

Reply:
Yeah, this is something I did here. I haven't been sure the best way to integrate this into Steven's merge process. https://github.com/minerllabs/minerl/compare/master...decodyng:limited_data_iterator?expand=1

Reply:
I think I'm going to put off the "integrate Cody's limited data iterator thing into Steven's Many Changes" task until later, and just use this `epoch_size` thing for now as a temporary change that enables tests to pass.

It will be easier for me to deal with that after I have all the dangling PRs successfully merged.